.github/workflows/install-slices.yaml

```diff
@@ -21,8 +21,14 @@ on:
 
 env:
   # Package architectures and chisel-releases branches to test on.
-  ARCHES: ${{ toJson('["amd64","arm64","armhf","i386","ppc64el","riscv64","s390x"]') }}
-  RELEASES: ${{ toJson('["ubuntu-20.04","ubuntu-22.04","ubuntu-24.04","ubuntu-24.10"]') }}
+  ARCHES: '["amd64","arm64","armhf","i386","ppc64el","riscv64","s390x"]'
+  RELEASES: |
+    [
+      {"ref": "ubuntu-20.04", "chisel-versions": ["v1.0.0","main"]},
+      {"ref": "ubuntu-22.04", "chisel-versions": ["v1.0.0","main"]},
+      {"ref": "ubuntu-24.04", "chisel-versions": ["v1.0.0","main"]},
+      {"ref": "ubuntu-24.10", "chisel-versions": ["v1.0.0","main"]}
```

Comment:
Why not test against the `latest` chisel version as well?

So the list would be something like `"latest", "v1.0.0", "main"`.

Reply:
That's captured here: https://github.com/canonical/chisel-releases/pull/509#discussion_r1981192703

Reply:
Ah I see. I will close this comment then, thank you.